services/cinema_hall.py

```diff
@@ -0,0 +1,18 @@
+from django.db.models import QuerySet
+from db.models import CinemaHall
+
+
+def get_cinema_halls() -> QuerySet:
```

Comment:
You could indicate that you return a `QuerySet` of `CinemaHall` instances in the return type annotation here and similar in other methods as well.